src/phpDocumentor/ApiReference/DocumentGroupDefinition.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\ApiReference;
+
+
+use League\Flysystem\FilesystemInterface;
+use phpDocumentor\DocumentGroupDefinition as DocumentGroupDefinitionInterface;
+use phpDocumentor\DocumentGroupFormat;
+
+final class DocumentGroupDefinition implements DocumentGroupDefinitionInterface
+{
+    /**
+     * @var DocumentGroupFormat
+     */
+    private $format;
+
+    /**
+     * @var FilesystemInterface
+     */
+    private $filesystem;
+
+    public function __construct(DocumentGroupFormat $format, FilesystemInterface $filesystem)
+    {
+        $this->format = $format;
+        $this->filesystem = $filesystem;
+    }
+
+    /**
+     * Returns the defined format.
+     *
+     * @return DocumentGroupFormat
+     */
+    public function getFormat()
+    {
+        return $this->format;
+    }
+
+    /**
+     * Returns the interal filesystem
+     *
+     * @return FilesystemInterface
+     */
+    public function getFilesystem()
+    {
+        return $this->filesystem;
+    }
+
+    /**
+     * Returns an array of paths to files to process.
+     *
+     * @return string[]
+     */
+    public function getFiles()
+    {
+        return [];
```

Comment:
Yes it is, I don't know yet how this will work. Most likely I will use the flyfinder plugin. But I didn't figure out yet how that works. And how it should be configured.